ZenPacks/zenoss/LinuxMonitor/FileSystem.py

```diff
@@ -85,10 +85,21 @@ def getRRDTemplateName(self):
 
     def getDefaultGraphDefs(self, drange=None):
         graphs = super(FileSystem, self).getDefaultGraphDefs(drange)
-        underlying = self.logicalVolume() or self.blockDevice()
-        if underlying:
-            for graph in underlying.getDefaultGraphDefs(drange):
-                graphs.append(graph)
+
+        # Explicitly re-label graphs displayed in other components for clarity
+        bd = self.logicalVolume() or self.blockDevice()
+        if bd:
+            for bdGraph in bd.getDefaultGraphDefs(drange):
+                if not bdGraph.get('title_override'):
+                    bdGraph = \
+                        {'title': '{0} ({1}: {2})'.format(bdGraph['title'],
+                                                         bd.__class__.__name__,
```

Comment:
We shouldn't expose literal class names to the user like this. I'd say either drop the type from the title, or use bd.class_label.

I have a style nitpick here too. Generally backslashes should be avoided unless they're really necessary to make the code read better. Indentation should also be consistent. I'd rather see this section look something like the following. It has advantages for keeping future diffs as small and precise as possible when this code needs updated.

```python
title = "{} ({}: {})".format(
    bdGraph["title"],
    bd.class_label,
    bd.titleOrId())

bdGraph = {
    "title": title,
    "url": bdGraph["url"],
    "title_override": True,
    }
```